python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -62,23 +63,33 @@ class MakeRawVisitInfo(object):
     However, the methods that transform units are less forgiving; they assume
     the user provides proper data types, since type errors in arguments to those
     are almost certainly due to coding mistakes.
+
+    Parameters
+    ----------
+    `lsst.log.Log`
+        Logger to use for messages.
+        (None to use ``Log.getLogger("MakeRawVisitInfo")``).
     """
 
     def __init__(self, log=None):
-        """Construct a MakeRawVisitInfo
-        """
         if log is None:
             log = Log.getLogger("MakeRawVisitInfo")
         self.log = log
 
     def __call__(self, md, exposureId):
-        """Construct a VisitInfo and strip associated data from the metadata
-
-        @param[in,out] md  metadata, as an lsst.daf.base.PropertyList or PropertySet;
-            items that are used are stripped from the metadata
-            (except TIMESYS, because it may apply to more than one other keyword).
-        @param[in] exposureId  exposure ID
-
+        """Construct a VisitInfo and strip associated data from the metadata.
+
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertyList` or `PropertySet`
+            Metadata to pull from.
+            Items that are used are stripped from the metadata (except TIMESYS,
+            because it may apply to more than one other keyword).
```

Comment:
Pleaser fix the existing grammatical error. The issue is that "TIMESYS" can apply to several date-related keywords, some of which may not yet have been processed. Perhaps use "because it may apply to other keywords".

Unfortunately this same issue appears several more times below. I hope you are willing to fix it everywhere.